Dialogs/Second_window_dialog.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from PyQt5 import uic
+from PyQt5 import QtWidgets
+
+
+class Second_window(QtWidgets.QDialog):
```

Comment:
Name of that window told me a lot about this class.

Reply:
Changed to Mapping_window